stylus-sdk/src/evm.rs

```diff
@@ -67,6 +92,6 @@ wrap_hostio!(
 wrap_hostio!(
     /// Gets the amount of ink remaining. See [`Ink and Gas`] for more information on Stylus's compute pricing.
     ///
-    /// [`Ink and Gas`]: https://developer.arbitrum.io/TODO
+    /// [`Ink and Gas`]: https://docs.arbitrum.io/TODO
```

Comment:
This page already exists, should be renamed to `Gas and Ink`
https://docs.arbitrum.io/stylus/concepts/stylus-gas